nightwatch.json

```diff
@@ -5,7 +5,13 @@
   "custom_assertions_path" : "",
   "page_objects_path" : "",
   "globals_path" : "",
-
+  "live_output" : false,
```

Comment:
We use live_output for ```Whether or not to buffer the output in case of parallel running. See below for details.```

`disable_colors` allows us to control whether or not to disable coloring of the cli output globally.

Reply:
please remove then, since its not valid for us.